src/Components/Tiles.components.js

```diff
@@ -117,7 +117,7 @@ export default class Tiles extends Component {
     render() {
         return (
             <div className="tiles pt-10">
-                <MDBContainer >
+                <MDBContainer style={{  marginLeft : "10vh", marginRight : "10vh"}} >
```

Comment:
Consider changing this to a CSS class. Inline styles are pointless if the style isn't going to change.